From a61a70605f9efc81fead5bf6984bc5ce39f1569d Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 27 May 2022 18:11:52 +0000
Subject: [PATCH] libstdc++: fix incomplete type of
 content::ContentRendererClient

Destructor of std::unique_ptr in libstdc++ uses sizeof() which
requires full definition of media::AudioEncoder for return type of
cast_streaming::ResourceProvider.
---
 content/public/renderer/content_renderer_client.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/content/public/renderer/content_renderer_client.cc b/content/public/renderer/content_renderer_client.cc
index 63456aa..637a2a7 100644
--- a/content/public/renderer/content_renderer_client.cc
+++ b/content/public/renderer/content_renderer_client.cc
@@ -6,6 +6,7 @@
 
 #include "base/command_line.h"
 #include "build/build_config.h"
+#include "components/cast_streaming/renderer/public/resource_provider.h"
 #include "content/public/common/content_switches.h"
 #include "media/base/demuxer.h"
 #include "media/base/renderer_factory.h"
-- 
2.35.1

